docs/CI-CD/GithubActions.md

```diff
@@ -28,33 +30,65 @@ echo $He_SP_PWD_CICD
+1. On GitHub, navigate to the main page of the repository.
+1. Under your repository name, click on the "Settings" tab.
+1. In the left sidebar, click Secrets.
+1. On the right bar, click on "Add a new secret"
+   ![alt text](../images/create-secret.png "GitHub Secrets")
+1. Type a name for your secret in the "Name" input box.
+1. Type the value for your secret.
+1. Click Add secret.
+   ![alt text](../images/Add-secret-name-value.png "Github Secret Details")
 
-1. Go to Settings -> Secrets
-2. Select "Add a new secret"
-3. Add the secret:
-    - Name: REGISTRY_USERNAME
-    - Value: {copy output from `echo $He_SP_ID_CICD` command}
-4. Add the secret:
-    - Name: REGISTRY_PASSWORD
-    - Value: {copy output from `echo $He_SP_PWD_CICD` command}
+The following secrets are required for the workflow to run:
 
-![alt text](../images/githubactions-secret.jpg "Add Github Secret")
+* DOCKER_USER  - your dockerhub user name
+* DOCKER_PAT   - the personal access token for dockerhub
+* SERVICE_PRINCIPAL  - your SP Client ID that has ACR Push IAM role - the value of $He_SP_ID_CICD
+* SERVICE_PRINCIPAL_PASSWORD   - the personal access token for the SP Client id - the value of $He_SP_PWD_CICD
+* TENANT  - The tenant ID of the subscription of the ACR - the value of $He_SP_TENANT_CICD
 
-## Add Github Workflow File (.yml)
+## Workflow Properties
+
+The workflow used in this project allows for images to be created for testing when a PR is created and Images for Production use when a Commit is tagged with a version via git tags.
+
+### Add Github Workflow File (.yml)
 
 1. Create a new branch in your forked repo off of master.
-2. Copy the [cicdWorkflow.yml](./cicdWorkflow.yml) from this folder to your forked repo in the .github/workflows directory. If the .github folder does not already exist at the root, create one.
-3. Update the yml file in your forked repo with the name of your ACR. The file defaults to "helium", so update with $He_Name from previous readme steps.
-    - The workflow is designed to push the same image twice to ACR, one with the latest tag and one tagged with the last commit SHA.  Update the tagging strategy as needed.
-    - It is also designed to be triggered on a push to the master branch, see [documentation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags) for more options.
-4. Merge changes into master through a pull request.  
+2. Copy the [dockerByPR.yml](./dockerByPR.yml) and the [dockerCI.yml](./dockerCI.yml) files from this folder to your forked repo in the .github/workflows directory. If the .github folder does not already exist at the root, create one.
+
+### Update dockerByPR
+
+1. On line 21, replace the name of your Docker Hub registry and image name you want to use. So for helium-csharp you could use `DOCKER_REPO: 'mydockerreg/helium-csharp'`
```

Comment:
do we have CSE feedback for the ability to store values in github?

I realize you can't see the value, but seems simpler to add this as a "secret" so you don't have to edit code.